prow/crier/controller.go

```diff
@@ -219,7 +223,7 @@ func (c *Controller) processNextItem() bool {
 
 	// assert the string out of the key (format `namespace/name`)
 	keyRaw := key.(string)
-	log := logrus.WithField("reporter", c.reporter.GetName()).WithField("prowjob", keyRaw)
+	log := logrus.WithField("reporter", c.reporter.GetName()).WithField("key", keyRaw)
```

Comment:
`keyraw` is `namespace/name` notation, so using it here as value for `prowjob` is just wrong